FWCore/ParameterSet/python/Config.py

```diff
@@ -554,6 +558,41 @@ def extend(self,other,items=()):
                 newSeq._place(name,self)
         self.__dict__['_Process__InExtendCall'] = False
 
+    # FIXME define explicitly if the items should be referenced or copied
```

Comment:
That was what I tried first - but I don't remember why it didn't seem to
work... I'll try again.